docs/README.md

```diff
@@ -1,106 +1,131 @@
+  - [code-server](https://github.com/coder/code-server)
+  - [JetBrains Projector](https://github.com/JetBrains/projector-server)
+  - [Jupyter](https://jupyter.org/)
+  - And others
 
-Coder workspaces don't stop at compute. You can add storage buckets, secrets, sidecars
-and whatever else Terraform lets you dream up.
+- Your existing remote development environment:
+  - [JetBrains Gateway](https://www.jetbrains.com/remote-development/gateway/)
+  - [VS Code Remote](https://code.visualstudio.com/docs/remote/ssh-tutorial)
+  - [Emacs](./ides/emacs-tramp)
 
-[Learn more about templates.](./admin/templates/README.md)
-
-## IDE Support
-
-You can use any [Web IDE](./admin/templates/extending-templates/web-ides.md) ([code-server](https://github.com/coder/code-server), [projector](https://github.com/JetBrains/projector-server), [Jupyter](https://jupyter.org), etc.), [JetBrains Gateway](https://www.jetbrains.com/remote-development/gateway/), [VS Code Remote](https://code.visualstudio.com/docs/remote/ssh-tutorial) or even a file sync such as [mutagen](https://mutagen.io/).
-
-<p align="center">
-  <img src="./images/ide-icons.svg" height=72>
-</p>
+- File sync such as [Mutagen](https://mutagen.io/)
 
 ## Why remote development
 
-Migrating from local developer machines to workspaces hosted by cloud services
-is an [increasingly common solution for
-developers](https://blog.alexellis.io/the-internet-is-my-computer/) and
-[organizations
-alike](https://slack.engineering/development-environments-at-slack). There are
-several benefits, including:
+Remote development offers several benefits for users and administrators, including:
 
-- **Increased speed:** Server-grade compute speeds up operations in software
-  development, such as IDE loading, code compilation and building, and the
-  running of large workloads (such as those for monolith or microservice
-  applications)
+- **Increased speed**
+  - Server-grade cloud hardware speeds up operations in software development, from
+  loading the IDE to compiling and building code, and running large workloads
+  such as those for monolith or microservice applications.
 
-- **Easier environment management:** Tools such as Terraform, nix, Docker,
-  devcontainers, and so on make developer onboarding and the troubleshooting of
-  development environments easier
+- **Easier environment management**
+  - Built-in infrastructure tools such as Terraform, nix, Docker, Dev Containers, and others make it easier to onboard developers with consistent environments.
 
-- **Increase security:** Centralize source code and other data onto private
-  servers or cloud services instead of local developer machines
+- **Increased security**
+  - Centralize source code and other data onto private servers or cloud services instead of local developers' machines.
+  - Provision and manage users with [SSO](https://coder.com/docs/admin/auth) and [Role-based access controlled (RBAC)](https://coder.com/docs/admin/rbac).
```

Comment:
@stirby @matifali can you confirm this for me, please?

Reply:
Instead of 

"Provision and manage users with SSO..."

Can we say

"Manage users and groups with SSO..."

Reply:
Otherwise looks good to me.